styles.css

```
@@ -10866,110 +10872,29 @@ body.modal-open {
-  background: #111827;
-  color: #f3f4f6;
-  transition: border-color 0.3s;
-}
+  text-decoration: none !important;
 
-.light-mode .search-input {
-  background: #ffffff;
-  color: #111827;
-  border-color: #d1d5db;
-}
+  background: var(--gradient-1) !important;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)